src/components/ConsoleComponent.js

```diff
@@ -24,71 +21,102 @@ const useStyles = makeStyles((theme) => ({
     boxSizing: 'border-box'
   },
   button: {
+    marginRight: '20px',
+    color: theme.palette.common.white,
+    textTransform: 'none'
+  },
+  problemsButton: {
     padding: 0,
     color: theme.palette.common.white,
     textTransform: 'none'
   },
   expandIcon: {
     width: '29px' // Lines up with padding
   },
-  warning: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'khaki'
-  },
-  error: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'red'
-  },
-  success: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'green'
-  },
   pre: {
     margin: '0px'
+  },
+  circle: {
+    'border-radius': '50%',
+    'min-width': '22px',
+    height: '22px',
+    paddingLeft: 1,
+    paddingRight: 1,
+    background: '#21121d',
+    'align-items': 'center',
+    display: 'flex',
+    marginLeft: 4,
+    'justify-content': 'center'
   }
 }));
 
 export default function Console(props) {
   const classes = useStyles();
+  const [problemsView, setProblemsView] = useState(false);
 
   const toggleExpandConsole = () => {
-    props.setExpandConsole(!props.expandConsole);
+    if (props.expandConsole && problemsView) {
+      setProblemsView(false);
+    } else {
+      props.setExpandConsole(!props.expandConsole);
+      setProblemsView(false);
+    }
+  };
+
+  const toggleProblemsConsole = () => {
+    if (props.expandConsole && !problemsView) {
+      setProblemsView(true);
+    } else {
+      setProblemsView(!problemsView);
+      props.setExpandConsole(!props.expandConsole);
+    }
   };
 
   return (
     <>
       <Box className={classes.consoleControls}>
-        <Button onClick={toggleExpandConsole} className={classes.button}>
+        <Button
+          onClick={toggleExpandConsole}
+          className={classes.button}
+          style={{
+            'text-decoration': props.expandConsole && !problemsView ? 'underline' : 'none'
+          }}
+        >
           {props.expandConsole ? (
             <ExpandMore className={classes.expandIcon} />
           ) : (
             <ExpandLess className={classes.expandIcon} />
           )}
           Console
-          <WarningIcon style={{ display: props.warningCount ? 'block' : 'none' }} className={classes.warning} />
-          {props.warningCount ? `${props.warningCount}` : ''}
-          <ErrorIcon style={{ display: props.errorCount ? 'block' : 'none' }} className={classes.error} />
-          {props.errorCount ? `${props.errorCount}` : ''}
-          <CheckIcon
-            className={classes.success}
+        </Button>
+        <Button onClick={toggleProblemsConsole} className={classes.problemsButton}>
+          <p
             style={{
-              display: props.consoleMessages.length > 0 && !props.warningCount && !props.errorCount ? 'block' : 'none'
+              'text-decoration': props.expandConsole && problemsView ? 'underline' : 'none',
```

Comment:
I wonder if the underline could be a little lower, more VSCode style:
![image](https://user-images.githubusercontent.com/52747882/134979230-9db7dfb4-a823-42df-b01b-7aa93aef128c.png)

That's just a small stylistic touch though, so feel free to ignore.

Reply:
I took this suggestion in commit `4388dc0` and it does look a lot better. Thanks!